### ISSUE TITLE
Multiple concurrent jrnl invocations lose entries

### ISSUE THREAD

**Issue by semi** [jrnl-org/jrnl#502]
```
$ jrnl -1
2017-09-21 15:00 my last entry

$ (echo first; sleep 1) | jrnl & echo second | jrnl
[Entry added to default journal]
[Entry added to default journal]
$ jrnl -2
2017-09-21 15:00 my last entry

2017-09-21 15:00 first
```

What happened to entry 'second'?

I just lost two day's worth of journal because I went to save and quit an entry that I had forgotten open in an editor.

Other than that, jrnl has been a real gem, thank you so much for developing it!

**Comment by sorsasampo** [comment]
I have been bitten by this as well. This is quite serious issue due to data loss.

**Comment by sorsasampo** [comment]
Keep this open. Data loss is bad.

**Comment by wren** [comment]
@sorsasampo Hi! The stalebot got a bit overzealous with the backlog. This issue should already be fixed in v2.0. 

Can you try the v2.0-rc2 version and verify if you still experience this issue? You can install with `pip install jrnl==2.0.0rc2`. 

Thanks!

**Comment by sorsasampo** [comment]
@wren, this particular issue seems to be fixed with 2.0.0rc3.post2.

**Comment by BenKettlewell** [comment]
I upgraded to the latest version but this isn't working as you'd expect.

I edited two entries from the same journal and made a couple changes. Saved and quit both and some, but not all the changes were retained.
